packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -400,7 +400,11 @@ abstract class FlutterCommand extends Command<void> {
         'mapping between the values and the original identifiers is stored in the '
         'symbol map created in the specified directory. For an app built with this '
         'flag, the \'flutter symbolize\' command with the right program '
-        'symbol file is required to obtain a human readable stack trace.',
+        'symbol file is required to obtain a human readable stack trace.\n\n'
+        'Because all identifiers are renamed methods like Object.runtimeType, '
```

Comment:
I think a comma between renamed and methods would make this clearer

Reply:
I think It  works better If  I remove the `like`

Reply:
The part that threw me was that were not talking about renamed methods

Reply:
Okay fixed for real